sufia-models/app/models/concerns/sufia/works/curation_concern/with_generic_files.rb

```diff
@@ -3,21 +3,6 @@ module CurationConcern
     module WithGenericFiles
       extend ActiveSupport::Concern
 
-      included do
-        # This used to have a hasFile relation when in hydra-works.  That does not seem to exist so I am using hasPart instead
-        has_many :generic_files, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.isPartOf, class_name: "GenericFile"
-
-        #has_many :generic_files, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.isPartOf
-        before_destroy :before_destroy_cleanup_generic_files
-
-        attr_accessor :files
-
-      end
-
-      def before_destroy_cleanup_generic_files
```

Comment:
Since a GenericFile in pcdm can be in more than one work are you sure that we do not still need the before destroy?  I do not see that behavior in hydra-works.